### DISCUSSION TITLE
Rendering resolution is always maxed up

### DISCUSSION THREAD

**Discussion by josepheiba** [PlayCover/PlayCover#896] [Q&A] [answered]
The rendering resolution is always maxed up to my native resolution even when I change it in settings, the settings only affect the size of the window but the rendering resolution is always 4k. Is this the normal behavior? I'm running Master Duel.

<img width="461" alt="Screenshot 2023-03-29 at 1 26 19 AM" src="https://user-images.githubusercontent.com/101673502/228402801-9482ffa9-a37d-46d7-9dc8-da7bdcb01b40.png">

**Comment by JoseMoreville** [comment, marked as answer]
Yes, I'm going to explain the least technical I can. it's rendering at 2x the resolution set by the user so it looks batter. Don't worry about it

**Reply by josepheiba** [reply]
thanks for the explanation, yes it looks better. Now I know how to control it thanks to your explanation. It would be helpful if I can directly control things, sometimes I want to play on battery power.
